sdk/storage/azure-storage-blob/azure/storage/blob/models.py

```diff
@@ -305,53 +305,45 @@ class ContainerPropertiesPaged(Paged):
         begin with the specified prefix.
     :param int results_per_page: The maximum number of container names to retrieve per
         call.
-    :param str marker: An opaque continuation token.
-    """
-    def __init__(self, command, prefix=None, results_per_page=None, marker=None):
-        super(ContainerPropertiesPaged, self).__init__(command, None)
+    :param str continuation_token: An opaque continuation token.
+    """
+    def __init__(self, command, prefix=None, results_per_page=None, continuation_token=None):
+        super(ContainerPropertiesPaged, self).__init__(
+            get_next=self._get_next_cb,
+            extract_data=self._extract_data_cb,
```

Comment:
Just curious, are there other places that use `PageIterator` compositionally?

Reply:
Answered on teams ;)